static/js/message_list_view.js

```diff
@@ -509,6 +509,7 @@ MessageListView.prototype = {
         if (stream_name !== undefined) {
             // If user narrows to a stream, doesn't update
             // trailing bookend if user is subscribed.
+            // Update when user is never subscribed(i.e. sub === undefined)
```

Comment:
(It seems like maybe we should revert the `message_list_view.js` hunk of https://github.com/zulip/zulip/commit/3d77aa49db3492f26accabda95cb84c3ae0069ce)